index.html

```diff
@@ -233,6 +234,16 @@ <h3>MathML Element Mappings</h3>
 		    <span class="subrole">AXSubrole: <code>TBD</code></span>
 		  </td>
 		</tr>
+                <tr id="el-math">
```

Comment:
nit: I wonder if we (should)care about proper indentation?

Reply:
mmh, I tried to make the indentation match with emacs but probably there were some tabs copied from HTML AAM...